assets/gitignore

```diff
@@ -0,0 +1,90 @@
+# This file contains default .gitignore rules. To use it, copy it to .gitignore,
```

Comment:
This asset is also provided by Drupal core. can we rely on the drupal core asset